packages/storybook/stories/va-alert.stories.jsx

```diff
@@ -177,6 +177,47 @@ const BackgroundOnlyTemplate = ({
   );
 };
 
+const I18nTemplate = ({
+  status,
+  'background-only': backgroundOnly,
+  'show-icon': showIcon,
+  'disable-analytics': disableAnalytics,
+  visible,
+  'full-width': fullWidth,
+  closeable,
+  headline,
+}) => {
+  const thisTemplate = useRef(null);
+  const [lang, setLang] = useState('en');
+
+  return (
+    <>
+      <button
+        onClick={() => {
+          const newLang = lang === 'en' ? 'es' : 'en';
+          setLang(newLang);
+          thisTemplate.current.closest('main').setAttribute('lang', newLang);
```

Comment:
You might have to update this to `document.querySelector('main').setAttribute('lang', lang);` instead of targeting the closest `main` element in Storybook. It looks like it's possible where multiple `main` elements exist on a page in Storybook when there are multiple examples for a component. I suspect our [mutation observer](https://github.com/department-of-veterans-affairs/component-library/blob/master/packages/core/src/i18n/i18n-setup.js) on `main` is different than the closest `main`.